src/lib/browserLocale.ts

```diff
@@ -45,6 +46,11 @@ function getPreferredLanguageMaybeWithRegion(): string {
   return languageWithRegion ?? baseLanguage // If none was found, return the base language
 }
 
+export function defaultBrowserCurrencyFormat(): CurrencyFormats | string {
+  const userLocale = getPreferredLanguageMaybeWithRegion()
+  return userLocale ? userLocale : CurrencyFormats.CommaDecimal
```

Comment:
Here we can actually use nullish coallescing

Reply:
Actually it's my bad, `getPreferredLanguageMaybeWithRegion` will always return a default value at the very least or whatever the browser returned, so we can just return its value directly. Also I've updated the "en-US" default to use the enum `CurrencyFormat.DotDecimal` instead (it's a rare case anyways,`navigator.languages ` and `navigator.language`  are rarely empty, the browsers which allow you to not set any language will still return the actual System default or browser install language in last resort, but it can theoretically be empty.